### PR TITLE
topology2: add ssp multi stream capture 

### PR DESCRIPTION
**Pull request by bardliao**
Add a multi capture stream on SSP port. This will cause IO error with "~/sof-test/test-case/multiple-pause-resume.sh -r 50" test.
Creating this PR for people who want to reproduce the issue.

### CODE REVIEW

**Comment by fredoh9** [comment]
SOFCI TEST

File: tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-module-copier-capture.conf [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+		#
+		unique	"instance"
+	}
+
+	Object.Widget {
+		copier."1" {
+			type dai_out
+			period_sink_count 2
+			period_source_count 0
+			node_type $HDA_LINK_INPUT_CLASS
```
Comment (RanderWang) [inline tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-module-copier-capture.conf:52]: why HDA_LINK_INPUT_CLASS ?

Comment (bardliao) [reply tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-module-copier-capture.conf:52]: > why HDA_LINK_INPUT_CLASS ?

This is copied from dai-copier-gain-mixin-capture.conf. It is a default value and will be overwrite in the .conf file where dai-copier-gain-module-copier-capture pipeline is used.

Comment (RanderWang) [reply tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-module-copier-capture.conf:52]: you can delete it to remove any confusing question.

Comment (bardliao) [reply tools/topology/topology2/include/pipelines/cavs/dai-copier-gain-module-copier-capture.conf]: > you can delete it to remove any confusing question.

ack, and removed

**Comment by ranj063** [comment]
@bardliao can we undraft this? All dependencies are merged now

**Review by lgirdwood — APPROVED** [review]
@ranj063 good for you ?

**Review by ranj063 — APPROVED** [review]

**Pull request merged**